app/listeners/hyrax_listener.rb

```diff
@@ -55,8 +55,9 @@ def on_collection_metadata_updated(event)
   # def on_collection_membership_update
   # end
 
-  # def on_file_characterized
-  # end
+  def on_file_characterized(event)
```

Comment:
Remind me, how does this get called again?

Reply:
Sure!  When the `'file.characterized'` gets published like in [here](https://github.com/samvera/hyrax/blob/double_combo/app/services/hyrax/characterization/valkyrie_characterization_service.rb#L19-L22), the listeners pick it up and then we can execute logic upon trigger, here's some comments that explain it better

https://github.com/samvera/hyrax/blob/double_combo/app/services/hyrax/listeners/file_listener.rb#L8-L13